src/main/java/pages/ShoesResultPage.java

```diff
@@ -0,0 +1,47 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShoesResultPage {
```

Comment:
Lets not categorise this page as `ShoeResultPage`. It can be applicable to any keyword other than shoes. Better naming convention would be `ResultPage`